src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * A record of ADD/DEL Persons
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.